src/main/resources/logback-spring.xml

```diff
@@ -1,46 +1,62 @@
+
+    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
+    <property name="LOG_PATTERN"
+              value="%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m | %mdc %n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}"/>
 
     <springProperty scope="context" name="appName" source="info.app.name"/>
 
-    <springProfile name="!logstash">
-        <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+    <springProfile name="dev || stdout">
+        <appender name="consoleAppender" class="ch.qos.logback.core.helpers.NOPAppender" />
+        <appender name="logAppender" class="ch.qos.logback.core.ConsoleAppender">
             <layout class="ch.qos.logback.classic.PatternLayout">
                 <Pattern>${LOG_PATTERN}</Pattern>
             </layout>
         </appender>
-        <appender name="aiAppender" class="ch.qos.logback.core.helpers.NOPAppender"/>
     </springProfile>
 
-    <springProfile name="logstash">
+    <springProfile name="!(dev || stdout)">
         <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
-            <encoder class="net.logstash.logback.encoder.LogstashEncoder">
-                <includeCallerInfo>true</includeCallerInfo>
-            </encoder>
+            <layout class="ch.qos.logback.classic.PatternLayout">
+                <Pattern>${LOG_PATTERN}</Pattern>
+            </layout>
         </appender>
-
-        <appender name="aiAppender" class="com.microsoft.applicationinsights.logback.ApplicationInsightsAppender"/>
+        <appender name="logAppender" class="com.microsoft.applicationinsights.logback.ApplicationInsightsAppender"/>
     </springProfile>
 
-    <logger name="uk.gov" additivity="false" level="DEBUG">
-        <appender-ref ref="consoleAppender"/>
-        <appender-ref ref="aiAppender"/>
+    <logger name="uk.gov.justice.hmpps" additivity="false" level="DEBUG">
+        <appender-ref ref="logAppender"/>
+    </logger>
+
+    <logger name="org.springframework" additivity="false" level="INFO">
+        <appender-ref ref="logAppender"/>
     </logger>
 
-    <logger name="org.springframework" additivity="false" level="INFO" >
+    <logger name="springfox.documentation.spring" additivity="false" level="INFO">
+        <appender-ref ref="logAppender"/>
+    </logger>
+
+    <logger name="com.microsoft.applicationinsights" additivity="false" level="INFO">
+        <appender-ref ref="logAppender"/>
+    </logger>
+
+    <logger name="com.zaxxer.hikari" additivity="false" level="INFO">
+        <appender-ref ref="logAppender"/>
+    </logger>
+
+    <logger name="org.apache.catalina" additivity="false" level="INFO">
+        <appender-ref ref="logAppender"/>
```

Comment:
Is some logging to stdout  still ok when not dev or stdout? These two loggers seem to do this

Reply:
How is its set at the moment we do get logging to stdout up to the point where the application has started 
So in the pods we only have the logs up to
`Started Application in x seconds (JVM running for x)`   then all the logs continue to go to app insights.